x-pack/legacy/plugins/task_manager/task_runner.ts

```diff
@@ -166,12 +166,23 @@ export class TaskManagerRunner implements TaskRunner {
    */
   public async claimOwnership(): Promise<boolean> {
     const VERSION_CONFLICT_STATUS = 409;
+    const attempts = this.instance.attempts + 1;
+    const now = new Date(Date.now());
+
+    const timeoutDate = intervalFromNow(this.definition.timeout!)!;
 
     try {
       this.instance = await this.store.update({
         ...this.instance,
         status: 'running',
-        runAt: intervalFromNow(this.definition.timeout)!,
+        startedAt: now,
+        attempts,
+        retryAt: new Date(
+          timeoutDate.getTime() +
+            (this.definition.getRetryDelay
+              ? this.definition.getRetryDelay(attempts, Boom.clientTimeout()) * 1000
+              : attempts * 5 * 60 * 1000) // incrementally backs off an extra 5m per failure
```

Comment:
probably worth making this a constant?  It's only in this PR twice tho ...

Reply:
Agreed, I have made the change for this one.